app/src/renderer/components/wallet/PageWallet.vue

```diff
@@ -83,6 +90,17 @@ export default {
       } else {
         return list
       }
+    },
+    stakedTokens() {
+      let tokens = 0
+      let values = Object.values(this.committedDelegations)
+      if (values.length > 0) {
+        tokens = values.reduce((sum, d) => sum + d)
+      }
+      return tokens
+    },
```

Comment:
Let's tighten that up:

```JavaScript
import _, { includes, orderBy } from "lodash"

stakedTokens() {
  return _.sum(Object.values(this.committedDelegations))
}
```

Reply:
Cool, I didn't realize there was a sum function :)